src/main/java/com/github/maxopoly/zeus/rabbit/incoming/artemis/SendPlayerTextComponentHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.github.maxopoly.zeus.rabbit.incoming.artemis;
+
+import com.github.maxopoly.zeus.ZeusMain;
+import com.github.maxopoly.zeus.model.GlobalPlayerData;
+import com.github.maxopoly.zeus.rabbit.DynamicRabbitMessage;
+import com.github.maxopoly.zeus.rabbit.incoming.GenericInteractiveRabbitCommand;
+import com.github.maxopoly.zeus.rabbit.outgoing.artemis.SendPlayerTextComponent;
+import com.github.maxopoly.zeus.servers.ConnectedServer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.json.JSONObject;
+
+public class SendPlayerTextComponentHandler extends GenericInteractiveRabbitCommand {
+    private enum FailureReason {
+        PLAYER_DOES_NOT_EXIST;
+    }
+
+    @Override
+    public void handleRequest(String ticket, ConnectedServer sendingServer, JSONObject data) {
+        UUID sender = UUID.fromString(data.getString("sender"));
+        UUID receiver = UUID.fromString(data.getString("receiver"));
+        String message = data.getString("message");
+        GlobalPlayerData receiverData = ZeusMain.getInstance().getPlayerManager().getOnlinePlayerData(receiver);
+        Map<String, Object> replyParameter = new HashMap<>();
+        if (receiverData == null) {
+            replyParameter.put("success", false);
+            replyParameter.put("reason", FailureReason.PLAYER_DOES_NOT_EXIST.toString());
+            sendReply(sendingServer, new DynamicRabbitMessage(ticket, "art_ans_send_player_text_comp", replyParameter));
+            return;
+        } else {
+            replyParameter.put("success", true);
+        }
+        sendReply(sendingServer, new DynamicRabbitMessage(ticket, "art_ans_send_player_text_comp", replyParameter));
+        sendReply(receiverData.getMCServer(), new SendPlayerTextComponent(ticket, sender, receiver, message));
```

Comment:
I'll admit I'm a bit confused about sendReply; it seems weird to me that sendReply exposes the Server. A "reply" is not a reply if you send it to a different server, haha! 

Still, this _appears_ to be a valid use; however, I'd recommend that in this case where you are forwarding a message to a different server via Zeus, you "reply" that it was sent, then "send" to target server via the RabbitGateway -- `Zeus.getInstance().getRabbitGateway().sendMessage(...)` (although that IS the lambda for interactiverabbitcommand, so again... 6 vs half dozen with API as written).

Reply:
I can update this to `ZeusMain.getInstance().getRabbitGateway().sendMessage(receiverData.getMCServer(), new SendPlayerTextComponent(ticket, sender, receiver, message));` and just leave a comment above saying this is the same as `sendReply()` however to improve readability we've done it this way if desired

Reply:
Nah, given how this is all structured, the sending server is hidden from the lambda until this handle() method is called anyway, so I'm not clear how it could be restructured meaningfully to properly bind the sendReply() with the sendingServer, so might as well leave it as is.

Just me being pedantic